test/scraping/cycling/procyclingstats/raceStartLists.test.js

```
@@ -1,33 +1,125 @@
+
+      const raceRiders = new RaceRiders();
+      await raceRiders.update(startlistData);
+    });
+
+    afterAll(async () => {
+      process.env.DATA_DIR = originalDataDir;
+      try {
+        await Bun.file(`${TEST_DATA_DIR}/teams.csv`).delete();
+        await Bun.file(`${TEST_DATA_DIR}/riders.csv`).delete();
+        await Bun.file(`${TEST_DATA_DIR}/raceRiders.csv`).delete();
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)